### PR TITLE
Move school experience pages

### PR DESCRIPTION
**Pull request by emmaachesongray**
### Trello card
https://trello.com/c/rrFCMtQy/6253-phase-1-move-get-school-experience-and-internships

### Context
To prepare for upcoming changes on 'Is teaching right for me?', we need to move the two school experiences pages out of this section.

### Changes proposed in this pull request
Move the following pages to the postgraduate teacher training section:
* teaching internships
* get school experience

### CODE REVIEW

**Review by Sarah-DfE — DISMISSED** [review]

**Comment by gemmadallmandfe** [comment]
@emmaachesongray @martyn-w This is now failing a unit test - can you have a look?

File: app/views/content/train-to-be-a-teacher/get-school-experience.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ calls_to_action:
         link_target: "https://schoolexperience.education.gov.uk/"
 promo_content:
     - content/train-to-be-a-teacher/promos/mailing-list-promo
-navigation: 5.50
+navigation: 20.20
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/get-school-experience.md:22]: I think you might need to be a bit careful here that you don't have conflicting numbers over the two PRs (quals and GSE)?

File: app/views/content/train-to-be-a-teacher/teaching-internships.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ promo_content:
   - content/train-to-be-a-teacher/promos/eta-promo-internships
 backlink: /
 fullwidth: true
-navigation: 5.55
+navigation: 20:25
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/teaching-internships.md:15]: I think you might need to be a bit careful here that you don't have conflicting numbers over the two PRs (quals and GSE)?

File: app/views/content/train-to-be-a-teacher/teaching-internships.md [hunk 3]
```diff
@@ -12,11 +12,11 @@ promo_content:
   - content/train-to-be-a-teacher/promos/eta-promo-internships
 backlink: /
 fullwidth: true
-navigation: 5.55
-navigation_title: Can I do a teaching internship?
+navigation: 20:25
```
Comment (martyn-w) [inline app/views/content/train-to-be-a-teacher/teaching-internships.md:15]: should this be `<number>.<number>` not `<number>:<number>`

**Review by martyn-w — APPROVED** [review]
👍  LGTM

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Pull request merged**